src/ImageService.Mappers/Db/DbImageMessageMapper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using LT.DigitalOffice.ImageService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.ImageService.Models.Db;
+using System;
+
+namespace LT.DigitalOffice.ImageService.Mappers.Db
+{
+    public class DbImageMessageMapper : IDbImageMessageMapper
+    {
+        public DbImageMessage Map(Guid parentId, string name, string content, string extention, Guid createdBy)
+        {
+            return new DbImageMessage()
```

Comment:
we use mapper for converting one type of entity to other, here you collect an entity from the fields, do you need a mapper here?

Reply:
will use entity from DigitalOffice.Models